src/main/java/ru/practicum/shareit/exception/handler/ErrorResponse.java

```diff
@@ -0,0 +1,11 @@
+package ru.practicum.shareit.exception.handler;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class ErrorResponse {
+
+    String massage;
```

Comment:
добавь приватный модификатор доступа и поправь опечатку
```suggestion
    String message;
```